homeassistant/components/person/__init__.py

```diff
@@ -50,7 +50,8 @@
 })
 
 CONFIG_SCHEMA = vol.Schema({
-    vol.Optional(DOMAIN): vol.Any(vol.All(cv.ensure_list, [PERSON_SCHEMA]), {})
+    vol.Optional(DOMAIN): vol.All(
+        cv.ensure_list, cv.remove_falsy, vol.Any([PERSON_SCHEMA], []))
```

Comment:
I don't think we need to accept an empty list specially. That will be allowed by default.